docs/getting-started/installing/installing-core-linux.md

```diff
@@ -122,17 +74,50 @@ The next thing you will need is a `project.json` file that will outline the depe
 
 ## Run your App
 
-You need to restore packages for your app, based on your project.json, with `dnu restore`.
+You need to restore packages for your app, based on your project.json, with `dotnet restore`.
```

Comment:
I would suggest extending the sentence above with something like:

> ...and execute the app using dotnet run

to account for the two commands you have specified below.